tools/install.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+# Copyright 2018 the Deno authors. All rights reserved. MIT license.
+from __future__ import print_function
+import os
+import json
+import sys
+import tempfile
+import shutil
+import gzip
+from zipfile import ZipFile
+import re
+try:
+    from urllib.request import urlopen
+except ImportError:
+    from urllib2 import urlopen
+
+GITHUB_URL = "https://github.com"
+releases_url_html = "https://github.com/denoland/deno/releases/latest"
+
+install_dir = os.path.join(tempfile.gettempdir(), "deno_install")
+
+home = os.path.expanduser("~")
+
+def get_latest_url():
+
+    res = urlopen(releases_url_html)
+
+    html = res.read().decode('utf-8')
+
+    urls = re.findall(r'href=[\'"]?([^\'" >]+)', html)
+
+    filename = {
+        "darwin": "deno_osx_x64.gz",
+        # python3 sys.platform returns linux ( python2 returns linux2 )
+        "linux": "deno_linux_x64.gz",
+        "linux2": "deno_linux_x64.gz",
+        "win32": "deno_win_x64.zip",
+        "cygwin": "deno_win_x64.zip"
+    }[sys.platform]
+
+    matching = [u for u in urls if filename in u]
+
+    if len(matching) != 1:
+        print("Bad download url")
+        print("urls", urls)
+        print("matching", matching)
+        sys.exit(1)
+
+    return GITHUB_URL + matching[0]
```

Comment:
I think it would be more readable if you 
```
return "https://github.com" + matching[0]
```

Reply:
Done!